samples/GenMintEntitlement.js

```diff
@@ -0,0 +1,91 @@
+const { ElvClient, Utils} = require("@eluvio/elv-client-js");
+const crypto = require("crypto");
+const { ethers } = require('ethers');
+const bs58 = require("bs58");
+
+let client;
+
+/* Sample configuration -- demov3 network */
+let tenant = "iten4TXq2en3qtu3JREnE5tSLRf9zLod"; // paladin
+let marketplaceObjectId = "iq__2dXeKyUVhpcsd1RM6xaC1jdeZpyr"; // A Place for Goats
+let sku = "C9Zct19CoEAZYWug9tyavX"; // Goat Pack One
```

Comment:
I would remove the comments referencing the names of test tenants and marketplaces.  It's fine to leave the IDs as examples.